<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutUser</class>
 <widget class="QDialog" name="ajoutUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utilisateur</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Agent</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="agentEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btAgent">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pseudo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pseudoEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mot de passe</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Profil</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="profilComboBox">
       <item>
        <property name="text">
         <string>Secrétariat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Caisse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chef Personnel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Admin</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Date création</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btValider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/19-Left-Arrow-icon.png</normaloff>:/icones/Icones/19-Left-Arrow-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>passwordEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>pseudoEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>agentEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>btAgent</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>ajoutUser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
